software/cross-cutting-concepts.qmd

```diff
@@ -277,22 +277,25 @@ Therefore, it is important to consider how they will be used.
 
 ## User permissions {#permissions}
```

Comment:
I think, ideally, this section would live somewhere else, but it didn't feel right to move it to User Requirements, as it has too much design/implementation detail for that. Maybe the nascent log-in section (as it's related to authorisation) or the security section?

Reply:
I agree. I see how this could fit into both a login or security section. Maybe it would be most logical to include it in the login section, since it's more directly related to that? And then link to it in the security section?

Reply:
I ended up moving this to Sprout as part of https://github.com/seedcase-project/design/issues/86